Add Release notes for 1.26 and contrib guides (#175)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/release_notes.adoc

> = What's new on {page-component-display-version}

Reviewer: Are you sure about that ? This will be overriden for each release ?
and so in `main`, we see 
```

Reviewer: I mean don't we want to keep history of each release in the release notes ?
I would be easier to people to search in one single release notes page

Reviewer: we would just preprend for each release to this page. what do you think ?
Something like 
```
= What's new in Serverless Workflow

== 1.26.0.Final
[...]

== 1.25.0.Final
[...]

...
```

Author: Yes, we will change for every release. Users can see in each branch what changed (navigating at the top of the page), rather than a huge asciidoc macaroni page.

Reviewer: suppose that I am a user and update to a newer version (let's say from 1.13 to 1.25) and something is broken ...
How to I look for that particular change in the release notes to know when it was changed ?
Do I have to go through all release notes pages ???

Author: Yes, you will. Ideally, we should have a Releases page in our GH repos. You can have a whole view of the releases, but not in the docs. 30 releases from now, the page will be huge, not ideal without paging. Docs are supposed to be short.

Author: See: https://github.com/quarkusio/quarkus/releases/tag/2.11.2.Final

And https://github.com/quarkusio/quarkus/releases.

Reviewer: fair enough

Author: We can add the links to previous release notes in this page, wdyt?

> = What's new on {page-component-display-version}

Reviewer 2: ```suggestion
= New features on {page-component-display-version}
```

:compat-mode!:

> == Notable Changes

Reviewer 2: ```suggestion
== Notable changes
```

> . link:https://issues.redhat.com/browse/KOGITO-7620[KOGITO-7620] - Improve events consumption thread pools configs

Reviewer 2: I'd suggest using bullet points instead of numbers. We use numbers when describing steps. 
```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7620[KOGITO-7620] - Improve events consumption thread pools configs
```

> . link:https://issues.redhat.com/browse/KOGITO-7612[KOGITO-7612] - Implement gRPC streaming scenarios

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7612[KOGITO-7612] - Implement gRPC streaming scenarios
```

> . link:https://issues.redhat.com/browse/KOGITO-7538[KOGITO-7538] - gRPC enum behaviour

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7538[KOGITO-7538] - gRPC enum behaviour
```

> == Other Changes and Bug Fixing

Reviewer 2: ```suggestion
== Other changes and Bug fixes
```

> . link:https://issues.redhat.com/browse/KOGITO-7761[KOGITO-7761] - Kogito SW Guides latest version are pointing to main branch

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7761[KOGITO-7761] - Kogito SW guides latest version is pointing to the main branch
```

> . link:https://issues.redhat.com/browse/KOGITO-7708[KOGITO-7708] - A few missing references and attributes are missing from the guides

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7708[KOGITO-7708] - A few missing references and attributes are missing from the guides
```

> . link:https://issues.redhat.com/browse/KOGITO-7694[KOGITO-7694] - [KSW Guides] KN CLI generates project with different name and extensions than other options

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7694[KOGITO-7694] - [KSW Guides] KN CLI generates a project with a different name and extensions than other options
```

> . link:https://issues.redhat.com/browse/KOGITO-7687[KOGITO-7687] - [KSW-Guides] - Restructure the guides to fit downstream structure

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7687[KOGITO-7687] - [KSW-Guides] - Restructure the guides to fit downstream structure
```

> . link:https://issues.redhat.com/browse/KOGITO-7636[KOGITO-7636] - [KSW-Guides] SW configuration properties guide

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7636[KOGITO-7636] - [KSW-Guides] SW configuration properties guide
```

> . link:https://issues.redhat.com/browse/KOGITO-7611[KOGITO-7611] - Remove test-utils from the SW examples and replace by Dev Services

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7611[KOGITO-7611] - Remove test-utils from the SW examples and replace by Dev Services
```

> . link:https://issues.redhat.com/browse/KOGITO-7610[KOGITO-7610] - [KSW-Guides] - Add How-To use RHBQ and Kogito productized artifacts

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7610[KOGITO-7610] - [KSW-Guides] - Add How-To use RHBQ and Kogito productized artifacts
```

> . link:https://issues.redhat.com/browse/KOGITO-7309[KOGITO-7309] - [KSW-Guides] Integration tests with PostgreSQL

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7309[KOGITO-7309] - [KSW-Guides] Integration tests with PostgreSQL
```

> . link:https://issues.redhat.com/browse/KOGITO-7301[KOGITO-7301] - [KSW-Guides] Persistence with PostgresSQL databases

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7301[KOGITO-7301] - [KSW-Guides] Persistence with PostgresSQL databases
```

> . link:https://issues.redhat.com/browse/KOGITO-7299[KOGITO-7299] - [KSW-Guides] Displaying workflow runtime data in serverless dashboards

Reviewer 2: ```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7299[KOGITO-7299] - [KSW-Guides] Displaying workflow runtime data in serverless dashboards
```
